File: Auto-generated-tasks/Difficulty-2/Task-8/Solution.py
from random import *
Num = 6
u = 0
A2 = 0
Mas = [0] * Num
for u in range(0, Num):
	Mas[u] = randint(4, 488)
	print("Mas[", u, "]=", Mas[u])
	A2 = A2 + Mas[u]
print("A2=", A2)

File: Auto-generated-tasks/Difficulty-2/Task-8/Solution.pas
Var
	Num := 6, u, A2 := 0: integer;
	Mas: Array[0..Num] of integer;
Begin
for u := 0 to Num do begin
	Mas[u] := Random(434) + 4;
	WriteLn("Mas[", u, "]=", Mas[u]);
	A2 := A2 + Mas[u];
end;
WriteLn("A2=", A2);
End.